MainWindow.xaml.cs

```diff
@@ -21,12 +21,15 @@ public partial class MainWindow : Window
         private MediaType? _currentMediaType { get; set; }
         private List<int> _driveSpeeds { get { return new List<int> { 1, 2, 3, 4, 6, 8, 12, 16, 20, 24, 32, 40, 44, 48, 52, 56, 72 }; } }
         private List<KeyValuePair<string, KnownSystem?>> _systems { get; set; }
-        private List<KeyValuePair<string, MediaType?>> _mediaTypes { get; set; }
-        private Process childProcess { get; set; }
+        private List<MediaType?> _mediaTypes { get; set; }
+        //private Process childProcess { get; set; }
```

Comment:
Any particular reason why childProcess is being kept around? Or will that be removed in a future update?